d3a_api_client/redis_aggregator.py

```diff
@@ -167,51 +167,31 @@ def batch_command(self, batch_command_dict, is_blocking=True):
 
     def _on_market_cycle(self, message):
         logging.info(f"A new market was created. Market information: {message}")
+        function_name = "on_market_cycle"
```

Comment:
I would very much prefer if you avoided selecting the methods using strings, and use closures instead. It is far more readable, and still one liner, to create a method `event_closure = lambda: self.on_market_cycle(message)`and pass the closure in the executor.submit method.

Reply:
Right , Thanks ! 
Adapted to your point